scripts/process_logs.yml

```diff
@@ -0,0 +1,53 @@
+
+input_logs:
+  - path: .                        # Where to look for log files
+    recursive: yes                 # Whether to descend into subdirectories recursively
+    pattern: Node(\d+)\.log.*      # Log file regex pattern to look for
+    node_group: 1                  # Group number that matches node identifier
+    only_timestamped: no           # Whether to discard non-timestamped lines
+
+
+chains:
+  main:
+  - log time:
+      monitoring: total
+  - process: warnings
+#  - match timestamp:
+#      min: 2018-03-15 10:14:10
+#      max: 2018-03-15 11:14:10
+  - process: tag_commits
+  - match tag: COMMIT
+  - log line: commits
+
+  warnings:
+  - match level: WARNING
+  - log time:
+      monitoring: warnings
+  - log line: warnings
+
+  tag_commits:
```

Comment:
It would be a good idea to have a list of prepared chain items for common operations such as ViewChange, CatchUp, Connection, etc.
It would be great to have these rules in a separate file so that we don't need to edit neither the script nor config if the format of some log entries changes.

Reply:
Yes, this is next in my TODO list